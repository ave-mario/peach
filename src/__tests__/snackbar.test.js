import React from 'react';
import renderer from 'react-test-renderer';
import { Wrapper } from 'shared/snackbar.css';
import 'jest-styled-components';

describe('Snackbar:', () => {
  it('component rednders correctly', () => {
    const wrapper = renderer.create(<Wrapper show />).toJSON();
    expect(wrapper).toMatchSnapshot();
  });
});
