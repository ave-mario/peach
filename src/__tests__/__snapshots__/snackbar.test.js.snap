// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snackbar: component rednders correctly 1`] = `
.c0 {
  visibility: hidden;
  max-width: 50px;
  height: 50px;
  margin: auto;
  background-color: #d31010;
  color: #fff;
  text-align: center;
  border-radius: 2px;
  position: fixed;
  z-index: 1;
  left: 0;
  right: 0;
  bottom: 20px;
  font-size: 17px;
  white-space: nowrap;
  visibility: visible;
  -webkit-animation: fadein 0.5s,expand 0.5s 0.5s,stay 3s 1s,shrink 0.5s 4s, fadeout 0.5s 4.5s;
  animation: fadein 0.5s,expand 0.5s 0.5s,stay 3s 1s,shrink 0.5s 4s, fadeout 0.5s 4.5s;
}

<div
  className="c0"
/>
`;
